share/spice/brainy_quote/brainy_quote.js

```diff
@@ -1,32 +1,89 @@
 (function(env) {
     env.ddg_spice_brainy_quote = function(api_result) {
 
+        Handlebars.registerHelper("getAuthor", function (arg1, arg2) {
+            if (arg1 == null && arg2 == null) {
+                return;
+            }
+            return arg1 == null ? arg2.replace(/ quote$/, "") : arg1;
+        });
+
         if (!api_result || api_result.error || api_result.author) {
             return Spice.failed('brainy_quote');
         }
-
-        Spice.add({
-            id: 'brainy_quote',
-            name: 'Quotations',
-            data: api_result,
-            meta: {
-                sourceName: 'Brainy Quote',
-                sourceUrl: api_result.source_url
-            },
-            signal: 'high',
-            normalize: function(item) {
-                return {
-                    person: item.header1.replace(/ quote$/, ""),
-                    url: item.source_url
-                };
-            },
-            templates: {
-                group: 'base',
-                options: {
-                    content: Spice.brainy_quote.content,
-                    moreAt: true
+	
+	if (!DDG.get_query().match(/quotes|quotations/)) {
+            Spice.add({
```

Comment:
We can move common parts from Spice object here and on [line 66](https://github.com/duckduckgo/zeroclickinfo-spice/pull/3288/files#diff-837555897925643aabbcf8c8d5071a1fR66) to a separate object above the plural check and only have customisations in `if` blocks before adding to Spice.